### PR TITLE
Fix mathjax placement in ternary aaxis titles

### PR DESCRIPTION
**Pull request by etpinard**
fixes https://github.com/plotly/plotly.js/issues/3455 -- cc @plotly/plotly_js 

I was really hoping to get rid of this hacky block 

https://github.com/plotly/plotly.js/blob/84e24aaf529000ec14fbe64aeeb4bcdc6ccac474/src/lib/svg_text_utils.js#L127-L142

entirely, but since our MathJax test coverage is lackluster, I chose the safe road.

### CODE REVIEW

**Comment by etpinard** [comment]
@antoinerg or @archmoj could one of review this thing at some point today?

Thank you!

File: src/lib/svg_text_utils.js [hunk 1]
```diff
@@ -134,7 +134,7 @@ exports.convertToTspans = function(_context, gd, _callback) {
                 else if(svgClass[0] === 'l') {
                     newSvg.attr({x: _context.attr('x'), y: dy - (newSvgH / 2)});
                 }
-                else if(svgClass[0] === 'a') {
+                else if(svgClass[0] === 'a' && svgClass.indexOf('atitle') === -1) {
```
Comment (archmoj) [inline src/lib/svg_text_utils.js]: Could we use 
<pre>
if(svgClass === 'atitle') {
</pre>
here?

Comment (etpinard) [reply src/lib/svg_text_utils.js]: Why would that be better?

Comment (archmoj) [reply src/lib/svg_text_utils.js]: I was wondering of an edge case where 'atitle' appears in other position in the text. 
Though I was not able find any word containing those letters in order!
Anyway could we use this condition instead to test if `atitle` only occurs at the start of the string?
<pre>
else if(svgClass[0] === 'a' && svgClass.indexOf('atitle') !== 0) {
</pre>

Comment (etpinard) [reply src/lib/svg_text_utils.js]: Ok. Done in -> https://github.com/plotly/plotly.js/pull/3513/commits/c7878b381dc5822faf20570abfa74a814297e4b1

**Comment by archmoj** [comment]
Thanks @etpinard for fixing this bug.
:dancer: when the tests were passed.
